src/esp/gfx/RenderCamera.cpp

```diff
@@ -149,10 +149,11 @@ uint32_t RenderCamera::draw(MagnumDrawableGroup& drawables, Flags flags) {
 
   if (flags & Flag::FrustumCulling) {
     // draw just the visible part
-    size_t numVisibles = cull(drawableTransforms);
+    previousNumVisibleDrawables_ = cull(drawableTransforms);
```

Comment:
Now this value "number of visible drawables" is a state stored in the class now. 

You will have to reset it at the beginning of this draw() function. Otherwise, if user turn off the FC during the run-time, this value is not updated.

`previousNumVisibleDrawables_ = 0;`

Reply:
I now reset this with the number of drawables in the draw call. Should be more accurate than 0, since all will be visible with no culling.